### ISSUE TITLE
`Error: Not Found` indicates that some expected artifacts are missing?

### ISSUE THREAD

**Issue by JimmyLv** [tauri-apps/tauri-action#435]
```shell
   Compiling state v0.5.3
    Finished release [optimized] target(s) in 7m 22s
    Bundling bibi-gpt_2.12.0_amd64.deb (/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/deb/bibi-gpt_2.12.0_amd64.deb)
    Bundling bibi-gpt_2.12.0_amd64.AppImage (/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage)
    Finished 2 bundles at:
        /home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/deb/bibi-gpt_2.12.0_amd64.deb
        /home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage

Expected artifacts paths:
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/deb/bibi-gpt_2.12.0_amd64.deb
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage.tar.gz
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage.tar.gz.sig
Found artifacts:
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/deb/bibi-gpt_2.12.0_amd64.deb
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage
Error: Not Found
```

**Comment by JimmyLv** [comment]
According to the output, these two expected artifacts are not found:

```
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage.tar.gz
/home/runner/work/BibiGPT/BibiGPT/src-tauri/target/release/bundle/appimage/bibi-gpt_2.12.0_amd64.AppImage.tar.gz.sig
```

Is there any way to not handle these additional artifacts paths? I remember that the previous version of `tauri-apps/tauri-action@dev` still built successfully?

**Comment by FabianLars** [comment]
I think I should change the wording from "Expected" to "possible"?
What it means is that the action looks at these paths for files (so that you can compare it with the files you would expect).
Above the "Expected paths" line is the actual Tauri output which also only lists 2 bundles which means that it was correct that the action detected 2 out of the 4 possible(expected) paths.

By the way the 2 "missing" artifacts will only be created if you enabled and configured Tauri's updater.

In other words, the action is running fine and your app was built correctly :)

**Comment by JimmyLv** [comment]
cool, I see. thanks a lot, so my build is successful, only because I configured TAURI_PRIVATE_KEY & TAURI_KEY_PASSWORD, thus the 2 "missing" artifacts path, is my understanding correct?

**Comment by FabianLars** [comment]
The action _always_ checks for the updater artifacts, it doesn't actually check if you configured the updater or not.

**Comment by JimmyLv** [comment]
cool, thanks!
